Replace log4j2 with slf4j and log4j backend

gedeck opened the pull request:
Replace log4j2 with slf4j and log4j backend

If an error occurs in the build path, remove this file
```
<workspacedirectory>/.metedata/.plugins/org.eclipse.jst.j2ee.web/webLibrariesCache.index 
```
and restart eclipse.

aclarkxyz commented:
I get this message when I run the template editor:

```
log4j:WARN No appenders could be found for logger (org.apache.jena.riot.system.stream.JenaIOEnvironment).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```

Need to figure out how to shut it up...

gedeck commented:
This is not a new warning message; I've seen `jena` complaining before even as we used log4j2. A solution is to provide the log4j configuration file location during startup. 
```
setenv PROPS ""
setenv PROPS "${PROPS} -Dlog4j.configuration=file:cfg/log4j.properties"

java ${PROPS} com.cdd.bae.main.Main $*
```

gedeck commented:
Added the `log4j.properties` file to the jar file. This switches off the WARN messages due to missing configuration.

Additional: Converted space indenting in build.xml to tabs; I highlight the additional line

gedeck commented on build.xml:
```diff
@@ -1,51 +1,52 @@
-			<fileset dir="." includes="data/ontology/**"/>
-            <fileset dir="." includes="images/*.png"/>
-			<zipgroupfileset dir="lib" includes="*.jar"/>
+		<copy file="cfg/log4j.properties" todir="bin" />
```
New line in `build.xml`
